frontend/dockerfile/dockerfile_test.go

```
@@ -3472,6 +3473,55 @@ RUN sh -c "[ $(cat /test5/foo) = 'hello' ]"
 	require.NoError(t, err)
 }
 
+func testAddURLChmod(t *testing.T, sb integration.Sandbox) {
+	skipDockerd(t, sb)
+	f := getFrontend(t, sb)
+	f.RequiresBuildctl(t)
+
+	resp := httpserver.Response{
+		Etag:    identity.NewID(),
+		Content: []byte("content1"),
+	}
+	server := httpserver.NewTestServer(map[string]httpserver.Response{
+		"/foo": resp,
+	})
+	defer server.Close()
+
+	dockerfile := []byte(fmt.Sprintf(`
+FROM busybox AS build
+ADD --chmod=755 %s /dest/
```

What's the default here? Could we use something more obscure? Eg. `0703`